File: norintegrate-mcp/src/main/java/com/norintegrate/mcp/NorIntegrateMcpApplication.java
package com.norintegrate.mcp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.persistence.autoconfigure.EntityScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication(scanBasePackages = "com.norintegrate")
@EntityScan("com.norintegrate.common")
@EnableJpaRepositories("com.norintegrate.common")
public class NorIntegrateMcpApplication {

  public static void main(String[] args) {
    SpringApplication.run(NorIntegrateMcpApplication.class, args);
  }
}

File: norintegrate-mcp/src/main/kotlin/com/norintegrate/mcp/NorIntegrateMcpApplication.kt
package com.norintegrate.mcp

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.persistence.autoconfigure.EntityScan
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@SpringBootApplication(scanBasePackages = ["com.norintegrate"])
@EntityScan("com.norintegrate.common")
@EnableJpaRepositories("com.norintegrate.common")
class NorIntegrateMcpApplication

fun main(args: Array<String>) {
    runApplication<NorIntegrateMcpApplication>(*args)
}
